src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -1101,6 +1102,27 @@ private string GetOriginalFileName(EpisodeFile episodeFile, bool useCurrentFilen
             return Path.GetFileNameWithoutExtension(episodeFile.RelativePath);
         }
 
+        private string GetReleaseHash(EpisodeFile episodeFile)
```

Comment:
This logic can all be done in the migration, then it doesn't need to be done at the time of naming. You could use what this PR does as an example. https://github.com/Sonarr/Sonarr/pull/656

Reply:
The db migration now performs the parsing. 
https://github.com/Sonarr/Sonarr/pull/6567/files#r1507044123

The timeout y'all discussed in the other PR may be an issue here as well, since it's also reparsing all titles.